### ISSUE TITLE
Interactive setup runs with no output

### ISSUE THREAD

**Issue by evwilkin** [figma/code-connect#222]
Please provide:

- Code Connect CLI version 1.2.4
- Operating system MacOS Sequoia 15.1.1
- Code Connect file, Figma design and/or relevant code snippet that could help us get more context
  - [Figma components file](https://www.figma.com/design/YqvMyyV4G347jSOgfYXi29/test-code-connect?node-id=3-7&m=dev)
  - [figma.config.json setup & draft PR](https://github.com/patternfly/patternfly-react/pull/11326/files#diff-5a693ae773c29dd4c357765c226444d46dd0285cb2786f1b1433a79ac786f3f4)

We have a monorepo with several packages, but after running `npx figma connect` from both the root of the project as well as the root of the package containing the components (`packages/react-core`) we provide all the requested information, wait a minute while the command executes, and then it completes but no mappings are found or files created.  Using `npx figma create connect ...` in each individual component directly to link it directly to a component/component set in the Figma file does work so we have been using that, but would prefer to take advantage of the automation if possible.

**Comment by karlpetersson** [comment]
Hey @evwilkin just to clarify, is the command you're trying to run `npx figma connect` (and not `npx figma create`)? E.g are you able to go through the wizard flow for connecting components, but no files are created?

**Comment by evwilkin** [comment]
Yes sorry about the typo! Correcting the original description now to reflect this

**Comment by slees-figma** [comment]
Hey @evwilkin! Thanks for the report. Please could you share either a screenshot or copy/paste the last bit of output of the CLI with the interactive setup? I cloned your repo and was able to complete the setup from `packages/react-core`, so would be helpful to know exactly which step is the problem. A couple more things to try:
- You can pass `--verbose` to see more detailed errors - though not sure this'll surface anything new
- We fixed a very similar bug a couple weeks ago in the last release (v1.2.4). It looks like you're on this anyway but worth triple checking as it sounds very similar!

**Comment by evwilkin** [comment]
Thanks for taking a look @slees-figma - here are two screenshots, the first run from the `/react-core` directory and the second from the same dir but passing `./src/components` as the components directory, both with the same effect (I did confirm the `v1.2.4` version seen in the first screenshot)
<img width="804" alt="Screenshot 2024-12-16 at 2 17 29 PM" src="https://github.com/user-attachments/assets/c3a039a7-dc18-46be-a687-2792ea4cca59" />

---

<img width="790" alt="Screenshot 2024-12-16 at 2 18 35 PM" src="https://github.com/user-attachments/assets/aa16ad90-d1cf-4687-99d8-61b668229d22" />

**Comment by evwilkin** [comment]
Also with the `--verbose` flag - I will note that initial run with this command without specifying the `./src/components` directory did give a warning about a file in `src/demos/...` (Parsing local files...Could not parse exports of file...):

---

<img width="802" alt="Screenshot 2024-12-16 at 2 24 46 PM" src="https://github.com/user-attachments/assets/8be433d4-7e59-44db-99e7-901eead91c1a" />

**Comment by slees-figma** [comment]
Many thanks @evwilkin! For the "What is the URL of the Figma file" you need to enter a Figma file URL (e.g. `https://www.figma.com/design/abc123/My-Design`). It looks like if that's missing the setup flow can't find any components and exits early. I'll take a look at improving the URL validation and also making sure we show an actual message here.

Closing for now but please reopen if you experience further issues.

**Comment by slees-figma** [comment]
Thanks @evwilkin. Looking at the code there is an error being thrown in parsing the response of the Figma file but I can't tell what it would be. I've asked for file access via your support ticket so I can investigate further.
